project3/index.html

```diff
@@ -0,0 +1,410 @@
+                    style="fill: rgb(255, 255, 255);" />
+                <circle cx="126" cy="12" r="2" style="fill: rgb(255, 255, 255);" />
+                <path
+                    d="M106.9599991,18.2798004c0.2900009,0,0.5800018-0.0601006,0.8597946-0.1900005  c1-0.4701996,1.4199066-1.669899,0.9502029-2.6601c-0.4799957-1-1.6704025-1.4296999-2.6704025-0.9497004  c-1,0.4697008-1.419899,1.6596012-0.9496994,2.6596012C105.5,17.8598995,106.2099991,18.2798004,106.9599991,18.2798004z"
+                    style="fill: rgb(255, 255, 255);" />
+                <path
+                    d="M106.139595,9.37988c0.2901001,0.1401396,0.5904083,0.20996,0.8804016,0.20996  c0.7397995,0,1.4496994-0.4101496,1.7997971-1.1201096c0.490303-0.9897504,0.080101-2.1899405-0.9100952-2.6796904  c-0.9897995-0.4902401-2.1900024-0.08008-2.6796951,0.9096699C104.7397003,7.68994,105.1498947,8.8896503,106.139595,9.37988z"
+                    style="fill: rgb(255, 255, 255);" />
+                <circle cx="116" cy="12" r="2" style="fill: rgb(255, 255, 255);" />
+                <circle cx="10" cy="76" r="2" style="fill: rgb(255, 255, 255);" />
+                <circle cx="10" cy="38" r="2" style="fill: rgb(255, 255, 255);" />
+                <path
+                    d="M12,46c0-1.1044998-0.8955002-2-2-2c-1.1044903,0-2,0.8955002-2,2v22c0,1.1044998,0.8955097,2,2,2  c1.1044998,0,2-0.8955002,2-2V46z"
+                    style="fill: rgb(255, 255, 255);" />
+                <circle cx="118" cy="76" r="2" style="fill: rgb(255, 255, 255);" />
+                <circle cx="118" cy="38" r="2" style="fill: rgb(255, 255, 255);" />
+                <path
+                    d="M120,46c0-1.1044998-0.8955002-2-2-2s-2,0.8955002-2,2v22c0,1.1044998,0.8955002,2,2,2s2-0.8955002,2-2V46z"
+                    style="fill: rgb(255, 255, 255);" />
+            </svg>
+            <label for="intelligence">
+                <div class="abilities ai">Intelligence</div>
+            </label>
+            <div class="selectbox">
+                <select name="rate" id="intelligence">
+                    <option value="norm">Within Human Norms</option>
+                    <option value="noticeable">Noticeably Above Norms</option>
+                    <option value="significant">Significantly Beyond Norms</option>
+                    <option value="wild">Wildy Beyond Norms</option>
+                    <option value="unlimited">Unlimited</option>
+                </select>
+            </div>
+
+
+
+
+
+            <div class="confirmbox"><input name="confirm" id="confirm" type="checkbox">
+                <span class="confirm"><label for="confirm">I confirm the above information represents my current
```

Comment:
I expect you can style the `label` directly and avoid the extra markup of the `span`